src/datasources/db/database.options.ts

```diff
@@ -10,4 +11,23 @@ export const dataSourceOptions: DataSourceOptions = {
   url: process.env.MIGRATIONS_DATABASE_URL,
   entities: [Webhook],
   migrations: [__dirname + '/../migrations/**/*{.ts,.js}'],
-};
+  ... (process.env.DB_SSL_ENABLE == 'true' ? { 
```

Comment:
I think the spread operator is not strictly required here. If I'm not wrong, the expression can be simplified:
```javascript
  ssl: process.env.DB_SSL_ENABLE == 'true',
  extra: process.env.DB_CA_PATH
    ? {
        ssl: {
          ca: readFileSync(process.env.DB_CA_PATH),
          rejectUnauthorized: true,
        },
      }
    : {
        ssl: {
          rejectUnauthorized: false,
        },
      },
```

Reply:
We can remove extra option and just keep ssl with the configuration. 
https://typeorm.io/data-source-options#postgres--cockroachdb-data-source-options